### DISCUSSION TITLE
两个自动切换夜间模式的选项能否合并为一个选项

### DISCUSSION THREAD

**Discussion by DeepChirp** [the1812/Bilibili-Evolved#1815] [功能建议 / Ideas]
您好！
https://github.com/the1812/Bilibili-Evolved/issues/932
既然这两个选项不能同时打开，那么我建议能否把这两个选项并到一起，名字叫做 `自动打开夜间模式` 之类的。打开该选项后，能够提供 `跟随系统` 和 `计划时间` 两个选项（可能做成卡片类型的要好看一些），选择计划时间的话再要求用户设定时间。

我觉得 `Dark Reader` 的设计可以参考一下，这个插件恰好也是实现切换主题功能的。~~该插件还多了一个根据坐标自动切换主题的功能，不过脚本实现起来有点麻烦。~~
<img width="276" alt="darkreader-automation" src="https://user-images.githubusercontent.com/66902050/117031245-f7a9c300-ad32-11eb-9b61-0113aa38aeb0.png">

如果您觉得这样比较麻烦的话，可以加判断语句，检测到一个打开另一个就变灰。但是我觉得这两个还是做成选项比较好。总之就是不要让这两个选项同时打开。

谢谢！

**Comment by the1812** [comment]
会想点办法处理这个的, 不过估计没`Dark Reader`这么精致(
